src/api/entitlements.js

```diff
@@ -294,8 +295,15 @@ export class Entitlement {
    * @param {!Array<string>} products
    * @param {string} subscriptionToken
    * @param {JsonObject|null|undefined} subscriptionTokenContents
+   * @param {!Timestamp} subscriptionTimestamp
```

Comment:
Must be nullable for things like metering entitlements where it makes no sense

Reply:
Fixed, thanks!